addons/test_mail/tests/test_mail_push.py

```diff
@@ -338,6 +339,49 @@ def test_notify_by_push_message_notify(self, push_to_end_point):
                     push_to_end_point.assert_not_called()
                 push_to_end_point.reset_mock()
 
+    @patch.object(odoo.addons.mail.models.mail_thread, 'push_to_end_point')
+    @mute_logger('odoo.tests')
+    def test_notify_call_invitation(self, push_to_end_point):
+        inviting_user = self.env['res.users'].sudo().create({'name': "Test User", 'login': 'test'})
+        channel = self.env['discuss.channel'].with_user(inviting_user).channel_get(
+            partners_to=[self.user_email.partner_id.id])
+        inviting_channel_member = channel.sudo().channel_member_ids.filtered(
+            lambda channel_member: channel_member.partner_id == inviting_user.partner_id)
+
+        inviting_channel_member._rtc_join_call()
+        push_to_end_point.assert_called_once()
+        payload_value = json.loads(push_to_end_point.call_args.kwargs['payload'])
+        self.assertEqual(
+            payload_value['title'],
+            "Incoming call",
+        )
+        options = payload_value['options']
+        self.assertTrue(options['requireInteraction'])
+        self.assertEqual(options['body'], f"Conference: {channel.name}")
+        self.assertEqual(options['actions'], [
+            {
+                "action": PUSH_NOTIFICATION_ACTION.DECLINE,
```

Comment:
In test, I'm not sure if we want to use the variable, as we want to assert what the client is expecting.
If we change the value only in python the test should fail and not pass.